### PR TITLE
test: adds unit tests for semantic utils

### PR DESCRIPTION
**Pull request by JBAhire**
## Description
- Unit tests for updated semantic utils. 

### Testing
Checked. 

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/messaging/OtelMessagingSemanticConventionUtilsTest.java [hunk 1]
```diff
@@ -81,4 +82,86 @@ public void testIsRabbitMqBackend() {
     v = MessagingSemanticConventionUtils.isRabbitMqBackend(e);
     Assertions.assertFalse(v);
   }
+
+  @Test
+  public void testGetRabbitmqDestination() {
```
Comment (skjindal93) [inline semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/messaging/OtelMessagingSemanticConventionUtilsTest.java:87]: Worth adding tests for negative cases as well, where the operation/destination does not exist

File: semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/messaging/OtelMessagingSemanticConventionUtilsTest.java [hunk 2]
```diff
@@ -81,4 +83,125 @@ public void testIsRabbitMqBackend() {
     v = MessagingSemanticConventionUtils.isRabbitMqBackend(e);
     Assertions.assertFalse(v);
   }
+
```
Comment (rish691) [inline semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/messaging/OtelMessagingSemanticConventionUtilsTest.java:86]: Can you rename this class to remove OTel prefix?

Comment (rish691) [reply semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/messaging/OtelMessagingSemanticConventionUtilsTest.java:86]: Also check for the prefix in the corresponding main class

**Review by rish691 — APPROVED** [review]
